frontend/src/modules/Core/Constants/Theme.ts

```diff
@@ -270,6 +270,11 @@ theme = createTheme(theme, {
         },
       },
     },
+    MuiTooltip: {
+      styleOverrides: {
+        tooltip: {},
+      },
+    },
```

Comment:
What's the purpose of this?

Reply:
I'll delete that. Was trying to change the max width of it so it didnt overlap with the edit icons